t1curve.py

```diff
@@ -0,0 +1,64 @@
+from pyspecdata.core import *
+import numpy
+import sympy
+
+class t1curve(fitdata):
+    def fitfunc_raw(self,p,x):
+        '''just the actual fit function to return the array y as a function of p and x'''
+        return p[0]+(p[1]-p[0])*(numpy.exp(-x/p[2]))
+    def fitfunc_raw_symb(self,p,x):
+        '''if I'm using a named function, I have to define separately in terms of sympy rather than numpy functions'''
+        return p[0]+(p[1]-p[0])*(sympy.exp(-x/p[2]))
+    #{{{ linfunc
+    def linfunc(self,x,y,xerr = None,yerr = None):
+        '''just the actual fit function to return the pair of arrays x',y' that should be linear
+        it accepts as inputs x and y, and it uses the output from the fit, where necessary
+        also optionally propagates the error based on yerr and xerr, which can be passed in to it
+        For the case of T1, we want to return ln(y-M(\infty)) = ln(M(0)-M(\infty)) - t/T_1
+        '''
+        #print 'DEBUG: y is',y
+        #print 'DEBUG: M(\infty) is',self.output(r'M(\infty)')
+        temp = self.output(r'M(\infty)')-y # the argument for log
+        #print 'DEBUG: temp is',temp
+        # note that there is some error associated with m(\infty) that I'm just taking for granted
```

Comment:
I think this whole file should be removed, since this was before Alec's updates?

Reply:
Yes, the whole file should be removed

Reply:
Just deleted it